arch/lkl/kernel/atomic.c

```diff
@@ -0,0 +1,86 @@
+#include <linux/kernel.h>
```

Comment:
agree. I actually wanted to do it in toolks/lkl/lib but was reluctant to add new lkl_host_ops entries for various atomic ops.  do you think it's nicer to do int host_ops or have better idea to do it in tools/lkl ?